server/services/fileManager.ts

```diff
@@ -0,0 +1,38 @@
+import b2 from 'backblaze-b2';
+import fs from 'fs';
+
+export default class FileManager {
+  protected b2: b2;
+
+  constructor() {
+    this.b2 = new b2({
+      applicationKeyId: process.env.BACKBLAZE_APP_KEY_ID as string,
+      applicationKey: process.env.BACKBLAZE_APP_KEY as string,
+    });
+    this.b2.authorize();
+  }
+  /**
+   * Upload the file.
+   *
+   * @param localFilePath - Path to the file on the temporary directory.
+   * @param fileName	- The name of the file.
+   * @return The location of the file on the file storage service
+   */
+  async upload(localFilePath: string, fileName: string) {
+    const uploadUrl = await this.b2.getUploadUrl({
+      bucketId: process.env.BACKBLAZE_BUCKET_ID as string,
+    });
+    await this.b2.uploadFile({
+      fileName: fileName,
+      data: fs.readFileSync(localFilePath),
+      uploadUrl: uploadUrl.data.uploadUrl,
+      uploadAuthToken: uploadUrl.data.authorizationToken,
+    });
+
+    return `${
+      process.env.ENV === 'production' ? 'https' : 'http'
```

Comment:
`PAGE_URL` on heroku already has `https://` and ends in a `/`.
 
i.e `PAGE_URL=https://university-board.herokuapp.com/` (we cannot change this or it will break code elsewhere)

Deploying this on heroku, thumbnails have the URL `http://https:/university-board.herokuapp.com//file/uoftbored/4f814bf9-ca55-480e-8260-59470a26527b.jpg`

No need to put `/` before file, or the `http` prefix.